src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java

```diff
@@ -0,0 +1,29 @@
+package com.onwelo.tinvalidator.validator;
+
+import java.util.regex.Pattern;
+
+public class ATValidator implements Validator {
+
+    private static final Pattern PATTERN = Pattern.compile("[0-9]{9}");
```

Comment:
Rozumiem że znaki `-` i `\` są wycinane gdzieś wyżej?

Reply:
in TinValidator class all not alphanumeric characters are removed